src/main/java/com/hoteldetails/pad/mappers/hoteldescription/response/DescriptionResponseMapper.java

```diff
@@ -42,15 +43,13 @@ public HotelDescriptionResponse map(final OTAHotelDescriptiveInfoRS response) {
                 safeSetProtoField(responseBuilder::setDescriptions, descriptionsMapper.map(content.getHotelInfo().getDescriptions()));
                 safeSetProtoField(responseBuilder::setServices, servicesMapper.map(content.getHotelInfo().getServices()));
                 safeSetProtoField(responseBuilder::setHotelItem, hotelItemMapper.map(content.getContactInfos(), content.getHotelInfo().getPosition()));
-                ResponseStatus.Builder reponseStatusBuilder = ResponseStatus.newBuilder();
-                safeSetProtoField(reponseStatusBuilder::setStatus, SUCCESS);
-                safeSetProtoField(responseBuilder::setResponseStatus, reponseStatusBuilder);
+                safeSetProtoField(responseStatusBuilder::setStatus, SUCCESS);
+                safeSetProtoField(responseBuilder::setResponseStatus, responseStatusBuilder);
             }
         } else {
-            ResponseStatus.Builder reponseStatusBuilder = ResponseStatus.newBuilder();
-            safeSetProtoField(reponseStatusBuilder::setStatus, FAILURE);
-            safeSetProtoField(reponseStatusBuilder::setErrorCode, response.getErrors().getError().get(0).getCode());
-            safeSetProtoField(reponseStatusBuilder::setErrorMessage, response.getErrors().getError().get(0).getValue());
+            safeSetProtoField(responseStatusBuilder::setStatus, FAILURE);
+            safeSetProtoField(responseStatusBuilder::setErrorCode, response.getErrors().getError().get(0).getCode());
+            safeSetProtoField(responseStatusBuilder::setErrorMessage, response.getErrors().getError().get(0).getValue());
```

Comment:
safeSetProtoField(responseBuilder::setResponseStatus, responseStatusBuilder);
this is missing here